### PR TITLE
Trying to fix windows buildbots after #74786

### PR DESCRIPTION
**Pull request by clayborg**
This patch fixes the SymbolFilePDBTests::TestMaxMatches(...) by making it test what it was testing before, see comments in the test case for details.

It also disables TestUniqueTypes4.py for now until we can debug or fix why it isn't working.

### CODE REVIEW

**Comment by DavidSpickett** [comment]
Thanks! Testing this locally.

If it works I'll probably land this and then work on what you suggested updating the PDB input file.

**Comment by DavidSpickett** [comment]
Works locally, I'll do the unit test update this afternoon and get the bot running again.

**Comment by DavidSpickett** [comment]
I didn't get time for the follow up, so I'm going to put the bot back online over the weekend and come back to it on Monday.

**Comment by DavidSpickett** [comment]
https://github.com/llvm/llvm-project/pull/75813 for the unit test.

**Comment by DavidSpickett** [comment]
On the subject of `TestUniqueTypes4.py`, the program on Windows contains no symbol data, but on Linux I see:
```
[   78]    111     Data            0x0000000000011038                    0x0000000000000008 0x00000021 ns::Foo<double>::value
```
Which I think is the key here.

The puzzling thing is that despite compiling with `-gdwarf` there is a PDB file produced. `llvm-dwarfdump` on the program file seems to show enough information for the lookup to succeed but lldb doesn't use it? Perhaps because it's thinking there is a pdb file or it's being told that the pdb file is in fact dwarf?

Basically, I'm not sure why a pdb file is generated at all here. So that may be *one* of the issues, because it doesn't explain why it worked previously.

Unless previously the lookup of `ns::FooDouble::value` did not require that there be a symbol present, but I don't see how else it would have found it.

**Comment by DavidSpickett** [comment]
The PDB was not the issue, it's always worked that way and the DWARF is still in the main executable.

I've opened https://github.com/llvm/llvm-project/issues/75936 for `TestUniqueTypes4.py` because I think this commit did not break the test that already existed, but in changing what the test looked for, uncovered a problem that was already there.
